rc/tools/go/gopls.kak

```diff
@@ -69,7 +69,7 @@ define-command -hidden -params 1 gopls-cmd %{
 define-command -hidden -params 0 gopls-def %{
     evaluate-commands %sh{
         jump=$( gopls definition "${kak_buffile}:${kak_cursor_line}:${kak_cursor_column}" 2> /dev/null \
-            |sed -e 's/-.*//; s/:/ /g; q' )
+            |sed -e 's/-[0-9]\+:.*//; s/:/ /g; q' )
```

Comment:
Nice fix, LGTM!

> Hope this is enough.

For all practical purposes, probably. Any filename that might fool the regex
(like `foo-0:.go`) is nigh illegal.

(If we needed to fix that, we would use the `-json` output option)